string[] joint_name
bool[]   torque
